### DISCUSSION TITLE
How to Install ComfyUI Manager on MACos?

### DISCUSSION THREAD

**Discussion by maomao123-pcu** [Comfy-Org/ComfyUI-Manager#223] [Q&A]
I have installed it according to the conventional method, and the terminal also shows that the installation is successful, but I cannot find the manager button after opening the website.
<img width="603" alt="截屏2023-12-08 13 24 16" src="https://github.com/ltdrdata/ComfyUI-Manager/assets/126594794/9c56388b-927e-40c6-bc0b-a3e890530e97">
<img width="1368" alt="截屏2023-12-08 13 24 35" src="https://github.com/ltdrdata/ComfyUI-Manager/assets/126594794/5a01ae82-1413-42cd-9124-540b047d44c9">

**Comment by ltdrdata** [comment]
This is not related to ComfyUI-Manager.
Your ComfyUI installation is invalid.

See below:
https://github.com/comfyanonymous/ComfyUI#apple-mac-silicon
